scripts/MAZE-EN/c100206002.lua

```diff
@@ -0,0 +1,66 @@
+--Shadow ghoul of the labyrinth
+--scripted by JoyJ
+function c100206002.initial_effect(c)
+	--search
+	local e1=Effect.CreateEffect(c)
+	e1:SetDescription(aux.Stringid(100206002,0))
+	e1:SetCategory(CATEGORY_TOHAND+CATEGORY_SEARCH)
+	e1:SetType(EFFECT_TYPE_IGNITION)
+	e1:SetRange(LOCATION_HAND)
+	e1:SetCountLimit(1,100206002)
+	e1:SetCost(c100206002.thcost)
+	e1:SetTarget(c100206002.thtg)
```

Comment:
redtext: `Parameter 2 should be "Function".`
```suggestion
	e1:SetTarget(c100206002.target)
```